loader_hub/web/knowledgebase_web/base.py

```diff
@@ -0,0 +1,101 @@
+from typing import List
+import time
+from playwright.sync_api import sync_playwright
```

Comment:
Loader-specific packages should be imported inside the class so that they're only imported when the class is instantiated or called. (Also, the tests won't pass otherwise because the package isn't installed)